arctic/store/version_store.py

```diff
@@ -524,9 +568,123 @@ def _publish_change(self, symbol, version):
             mongo_retry(self._changes.insert_one)(version)
 
     @mongo_retry
-    def write(self, symbol, data, metadata=None, prune_previous_version=True, **kwargs):
+    def _write_metadata_history(self, symbol, metadata, start_time, end_time=None, **kwargs):
```

Comment:
`metadata_history` makes it sound like it's writting multiple metadata entries.  Perhaps `_write_metdata_entry`?